geode-web-api/src/main/java/org/apache/geode/rest/internal/web/security/RestSecurityConfiguration.java

```diff
@@ -50,7 +50,7 @@ public AuthenticationManager authenticationManagerBean() throws Exception {
   protected void configure(HttpSecurity http) throws Exception {
     http.sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS).and()
         .authorizeRequests()
-        .antMatchers("/ping", "/docs/**", "/swagger-ui.html", "/experimental/api-docs/**",
+        .antMatchers("/ping", "/docs/**", "/swagger-ui.html", "/v1/api-docs/**",
```

Comment:
Does this need to be versioned? Should it be more like the "docs" one and just be "/api-docs/**"?

Reply:
there is no difference for now, since we only have one version of dev rest.